README.md

```diff
@@ -127,10 +127,18 @@ your project:
 	  	config: 'geo_cfg.yml'
 	  }
 
+### Custom logic to skip the redirection
+
+You can pass a block to the `:skip_if` option to provide custom logic to skip the redirection.
+
+    Rails.application.middleware.use GeoRedirect::Middleware, skip_if: ->(req) { req.bot? }
+
+will ignore requests from bots (the code inside the block is just pseudo-code).
+
 ### Debugging
 
 You can add a `logfile` path string when adding the middleware if you want it to
-log some of its decision process into the file.  
```

Comment:
Yes, I'm familiar with such editor setups from some of my co-workers :)
If like them you're using Sublime, here's what you want: http://robandlauren.com/2013/11/21/configuring-sublime-text-markdown/